mapflow/functional/service/data_catalog.py

```diff
@@ -160,3 +183,15 @@ def selected_mosaic(self) -> Optional[MosaicReturnSchema]:
         if not first:
             return None
         return first[0]
+    
+    def selected_images(self, limit=None) -> List[MosaicReturnSchema]:
+        ids = self.view.selected_images_ids(limit=limit)
+        print(ids)
```

Comment:
print!

Reply:
Yes, these are the same changes related to "selected images" as in other branches. 
I  will delete this whole function including print when making a new branch to merge without conflicts